libbeat/beat/beat.go

```diff
@@ -149,6 +151,7 @@ func (b *Beat) Start() error {
 	// Additional command line args are used to overwrite config options
 	err, exit := b.CommandLineSetup()
 	if err != nil {
+		logp.Err("fails to load command line setup")
```

Comment:
No error were written here to the log as the errors are returned. It Should be part of the implementing beat to decide if the error should be written to log or not.